climada/hazard/centroids/centr.py

```diff
@@ -79,280 +52,215 @@ class Centroids():
 
     Attributes
     ----------
-    meta : dict, optional
-        rasterio meta dictionary containing raster properties: width, height, crs and transform
-        must be present at least. The affine ransformation needs to be shearless (only stretching)
-        and have positive x- and negative y-orientation.
-    lat : np.array, optional
-        latitude of size size
-    lon : np.array, optional
-        longitude of size size
-    geometry : gpd.GeoSeries, optional
-        contains lat and lon crs. Might contain geometry points for lat and lon
-    area_pixel : np.array, optional
-        area of size size
-    dist_coast : np.array, optional
-        distance to coast of size size
-    on_land : np.array, optional
-        on land (True) and on sea (False) of size size
+    lat : np.array
+        latitudes in the chosen crs (can be any unit)
+    lon : np.array
+        longitudes in the chosen crs (can be any unit)
+    crs : str, optional
+        coordinate reference system, default is WGS84
     region_id : np.array, optional
-        country region code of size size
-    elevation : np.array, optional
-        elevation of size size
+        region numeric codes
+        (can be any values, admin0, admin1, custom values)
+    on_land : np.array, optional
+        on land (True) and on sea (False)
     """
 
-    vars_check = {'lat', 'lon', 'geometry', 'area_pixel', 'dist_coast',
-                  'on_land', 'region_id', 'elevation'}
-    """Variables whose size will be checked"""
-
     def __init__(
         self,
-        lat: Optional[np.ndarray] = None,
-        lon: Optional[np.ndarray] = None,
-        geometry: Optional[gpd.GeoSeries] = None,
-        meta: Optional[Dict[Any, Any]] = None,
-        area_pixel: Optional[np.ndarray] = None,
-        on_land: Optional[np.ndarray] = None,
-        region_id: Optional[np.ndarray] = None,
-        elevation: Optional[np.ndarray] = None,
-        dist_coast: Optional[np.ndarray] = None
+        *,
+        lat: Union[np.ndarray, list],
+        lon: Union[np.ndarray, list],
```

Comment:
List of what? Maybe add type hint here (float)
```suggestion
        lat: Union[np.ndarray, list[float]],
        lon: Union[np.ndarray, list[float]],
```